MapboxDirections/MBVisualInstructionBanner.swift

```diff
@@ -21,6 +21,13 @@ open class VisualInstructionBanner: NSObject, NSSecureCoding {
      */
     @objc public let secondaryInstruction: VisualInstruction?
     
+    /**
+     A visual instruction that is presented simultaneously to provide information about an additional maneuver that occurs in rapid succession.
+     
+     This instruction could either contain the visual layout information or the lane information about the next maneuver after the upcoming maneuver.
```

Comment:
`This instruction could either contain the visual layout information or the lane information about the upcoming maneuver.`